bitnami/wordpress/values.yaml

```diff
@@ -450,18 +457,22 @@ ingress:
 
   ## The list of additional hostnames to be covered with this ingress record.
   ## Most likely the hostname above will be enough, but in the event more hosts are needed, this is an array
+  ##
   ## extraHosts:
-  ## - name: wordpress.local
-  ##   path: /
+  ##   - name: wordpress.local
+  ##     path: /
   ##
 
   ## Any additional arbitrary paths that may need to be added to the ingress under the main host.
   ## For example: The ALB ingress controller requires a special rule for handling SSL redirection.
   ## extraPaths:
   ## - path: /*
+  ##   pathType: ImplementationSpecific
```

Comment:
Isn't `pathType` / `ImplementationSpecific` something only added very recently (in 1.18)? https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/

It would break compatibility with this chart and older K8s versions where it was working before, even if you specify an older value to `ingress.apiVersion`.

Reply:
Fixed